config/routes.rb

```diff
@@ -1,5 +1,5 @@
 if ShopifyApp.configuration.routes_enabled?
-  ShopifyApp::Engine.routes.draw do
+  Rails.application.routes.draw do
```

Comment:
Okay I'll make the default in the config to false but then set it in the
initializer so it's a visible setting - how does that sound?
On May 13, 2015 6:14 PM, "Steve Lounsbury" <notifications@github.com> wrote:

> In config/routes.rb
> <https://github.com/Shopify/shopify_app/pull/133#discussion_r30281633>:
>
> > @@ -1,5 +1,5 @@
> >  if ShopifyApp.configuration.routes_enabled?
> > -  ShopifyApp::Engine.routes.draw do
> > +  Rails.application.routes.draw do
>
> I kinda like that, or make the default config for routes_enabled false so
> they have to opt-in?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Shopify/shopify_app/pull/133/files#r30281633>.
>